e2e/opensea-api-mock/src/utils/logger.ts

```diff
@@ -0,0 +1,26 @@
+import winston, { format } from "winston";
```

Comment:
why both winston and morgan? and not just one?

Reply:
Interesting question. 
Same code as in meta-tx-gateway.
I'd say morgan is just used to format the HTTP requests, seems ideal for that.
While Winston is used to log out globally (not only at HTTP level).
Morgan redirect to Winston anyway, so it doesn't really appear to be redundant.